web/urls.py

```diff
@@ -45,6 +40,25 @@
     prefix_default_language=False,
 )
 
+# Configure login based on if we have configured Dataporten or not.
+if settings.SOCIAL_AUTH_DATAPORTEN_SECRET:
+    urlpatterns += i18n_patterns(
+        path('login/', RedirectView.as_view(url='/login/dataporten/', query_string=True), name='login'),
+        path('logout/', Logout.as_view(), name='logout'),
+        re_path(r'^complete/(?P<backend>[^/]+){0}$'.format(extra), login_wrapper),
+        path('', include('social_django.urls', namespace='social')),
+        prefix_default_language=False,
+    )
+
+# If it is not configured, we would like to have a simple login page. So that
```

Comment:
I would prefer if this comment was inside the else block. Now the if and else look detached

Reply:
I agree that it becomes more readable, have moved it now.